plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -274,7 +276,10 @@ class GeneralServiceFormSection extends Component {
             </FieldError>
           </FormGroup>
 
-          <FormGroup className="flex flex-item-align-end column-2 flush-left">
+          <FormGroup className={{
```

Comment:
Ahh, yes I agree @nLight - thought @philipnrmn had discovered some React wizardry (it would be nice if it was native). I think we should avoid this pattern too if we're enabling it in another component.